apps/backend/src/event/dto/create-event.dto.ts

```diff
@@ -0,0 +1,5 @@
+import { OmitType } from '@nestjs/swagger';
+
+import { Event } from '../entities/event.entity';
+
+export class CreateEventDto extends OmitType(Event, ['ownerGroupId', 'ownerUserId']) {}
```

Comment:
Omit only id, because that's the only information the database comes up with alone. The owner we should specify.